src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
```

Comment:
you can use de-structuring here to make this more readable. `const {transaction} = props`